### PR TITLE
Removed Fields from Create Circle Page

### LINKED ISSUE #406: Improve onboarding for permissionless circle creation
- Cut 3 questions
- Change wording

From @ivy 's report:
image

“How may we contact you?” -> “Circle Point of Contact.”

### PR DESCRIPTION
**Pull request by teeolendo**
closes #406 I removed the following fields.

- What do you want to use Coordinape for?
- How many people will be in the circle?
- How much will you distribute each month?

Fixes #406

### CODE REVIEW

File: hardhat/deploymentInfo.json [hunk 1]
```diff
@@ -30,13 +30,13 @@
       "address": "0xD369D1481B157D1d791F78766660f5ddF715f1f2"
     },
     "ApeRouter": {
-      "address": "0x3D24345bb7FEE3B0B50d66a58946722BBbB26dFf"
+      "address": "0x42c7FfF180Ce8ef5414400696d3BcdD8A6b9354b"
```
Comment (crabsinger) [inline hardhat/deploymentInfo.json:33]: Hmm, maybe move this change to a separate PR that describes the update to the address...

**Review by crabsinger — CHANGES_REQUESTED** [review]
@teeolendo the PR looks good, except I'd separate out the deploy address changes, if we intend to keep that in.

**Comment by crabsinger** [comment]
I also added "fixes #406" to description to trigger autolinking between issue and PR, doesn't seem like it got linked just from the "#406" but needs the "fixes #406". 

https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

File: hardhat/deploymentInfo.json (hunk 1 above)
Comment (levity) [reply hardhat/deploymentInfo.json:33]: These changes can be thrown away, & should stop happening now that I fixed the issue in setup-hardhat.sh that was causing them

Comment (teeolendo) [reply hardhat/deploymentInfo.json:33]: Yeah, this isn't a material change. I'll resolve this.

**Review by teeolendo — COMMENTED** [review]
This isn't a material change and can be ignored.

**Review by levity — APPROVED** [review]

**Comment by levity** [comment]
@teeolendo ordinarily i'd ask you to remove the changes from the PR, but in this case it was pretty quick for me to just do it myself. i'll let you do the merge yourself (first PR merge! 🎉 )

**Review by exrhizo — APPROVED** [review]
LGTM, I tried the vercel preview
